math/geometry/distance_test.go

```diff
@@ -0,0 +1,104 @@
+package geometry
+
+import (
+	"testing"
+)
+
+type args struct {
+	p1 EuclideanPoint
+	p2 EuclideanPoint
+}
+
+func TestFindDistanceBetweenTwoPoints(t *testing.T) {
+	tests := []struct {
+		name    string
+		args    args
+		want    float64
+		wantErr bool
```

Comment:
None of your tests are using this. This method is used to test whether you get the error you expect. Please write an additional case to test for `ErrDimMismatch`.

Reply:
Noted & done.